src/formatDate.js

```diff
@@ -50,7 +50,74 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let year;
+  let month;
+  let day;
+  let separator = fromFormat[3];
+  const fromArray = date.split(separator);
```

Comment:
on my opninion, there is no need to declare separator
```suggestion
  const fromArray = date.split(fromFormat[3]);
```

Reply:
yeah, it seems so. but it's more understandable for me using a separate variable for it)) thanks!